dual_net.py

```diff
@@ -81,6 +83,8 @@ def bootstrap(self):
             tf.train.Saver().save(sess, self.save_file)
 
     def train(self, tf_records, init_from=None, logdir=None, num_steps=None):
+        def should_log(i):
+            return logdir is not None and i % 100 == 0
```

Comment:
Make log sample a constant?

Reply:
+1 to this, extracting to a constant would also make it monkeypatchable for local_rl_loop or etc.

Reply:
are we talking about making 100 a parameter `log_every` or something like that?

Reply:
SUMMARY_FREQUENCY or whatever.  not really needed.